influx/templates_test.go

```diff
@@ -274,7 +274,7 @@ func TestGroupByVarString(t *testing.T) {
 				ReportingInterval: 10 * time.Second,
 				Duration:          24 * time.Hour,
 			},
-			want: "time(369s)",
+			want: "time(370s)",
 		},
 		{
 			name: "String() outputs a minimum of 1s intervals",
```

Comment:
this test name should be renamed to something like "outputs milliseconds if interval less than one second"